beacon_node/operation_pool/src/lib.rs

```diff
@@ -312,20 +309,26 @@ impl<T: EthSpec> OperationPool<T> {
     }
 
     /// Prune if validator has already exited at the last finalized state.
-    pub fn prune_voluntary_exits(&self, finalized_state: &BeaconState<T>) {
+    pub fn prune_voluntary_exits(&self, finalized_state: &BeaconState<T>, spec: &ChainSpec) {
         prune_validator_hash_map(
             &mut self.voluntary_exits.write(),
-            |validator| validator.is_exited_at(finalized_state.current_epoch()),
+            |validator| validator.exit_epoch != spec.far_future_epoch,
             finalized_state,
         );
     }
 
     /// Prune all types of transactions given the latest finalized state and head fork.
```

Comment:
I guess this comment is technically outdated.

Reply:
Hmm, true. I'll save the CI mins and just merge tho